lib/template/lib/routes.rb

```diff
@@ -4,7 +4,7 @@
   use Pliny::Middleware::CORS
   use Pliny::Middleware::RequestID
   use Pliny::Middleware::RequestStore, store: Pliny::RequestStore
-  use Pliny::Middleware::Timeout, timeout: Config.timeout if Config.timeout > 0
+  use Rack::Timeout if Config.timeout > 0
```

Comment:
Also, @kch, you should consider taking options into the middleware's constructor! That would allow for varying timeout depending on the middleware stack that you're building (compared to a global option).